shared/kv/cachedAuthData.ts

```diff
@@ -0,0 +1,13 @@
+import {SalesforceAuthCache} from "@/shared/types/salesforceTypes";
+import {kv} from "@vercel/kv";
+
+export const getCachedAuthData=async(refreshToken:string)=>{
+    return await kv.get(refreshToken) as SalesforceAuthCache
```

Comment:
Check all your `kv.get` if they can be swapped for more efficient `kv.hset` or other redis key value funcs - there's a few other types that might make more sense